packages/g/gromacs/package.yml

```diff
@@ -14,6 +14,7 @@ optimize   : avx256
 builddeps  :
     - pkgconfig(fftw3f)
     - pkgconfig(lapack)
+    - pkgconfig(muparser)
     - pkgconfig(openblas)
     - pkgconfig(x11)
```

Comment:
Hi!

Sorry for barging in, but, FYI, GROMACS dropped the last X11 tool a few releases ago, so `pkgconfig(x11)` and `-DGMX_X11=1` below can be removed since they would have no effect anyway.

Also, looks like GROMACS is being built with debug symbols. Since it's typically used in performance-sensitive contexts, perhaps it is better to build with `-DCMAKE_BUILD_TYPE=Release` (as long as it fits Solus's policies)?

Reply:
Thanks for the info! Going to take a look at that later.

RE:  build type: The Solus macros generally use `-DCMAKE_BUILD_TYPE=RelWithDebInfo`, and then the debug symbols are stripped into their own package. If there is a significant performance penalty I guess this could be changed for this particular package.

Reply:
> If there is a significant performance penalty I guess this could be changed for this particular package.

On i9-12900K with GCC 11, the performance difference between `-DCMAKE_BUILD_TYPE=RelWithDebInfo` and `-DCMAKE_BUILD_TYPE=Release` is around 5%. If there is an established policy of using `RelWithDebInfo`, the difference is likely not worth breaking it.

Reply:
I mean, 5% is not too shabby in my opinion. Packages have been optimized for less.

But as this PR has already been merged I'll leave things as they are for now and made a note for the next update to definitely remove the redundant X11 stuff, and have another think about removing the debuginfo.

Reply:
Solus is already doing quite well by using x86-64-v2 / AVX2, and not SSE4.1 (like, e.g., Debian -- not blaming them, but it is not great for performance).

But, realistically, nowadays, most large scale molecular dynamics runs are on GPUs and/or on multi-node clusters; for both, making portable binaries is not realistic. We're exploring a solution for GPUs with [AdaptiveCpp](https://github.com/AdaptiveCpp/AdaptiveCpp/), but ACpp itself is not too widespread yet. So, 5% of CPU performance is good, but perhaps not critical.

I definitely don't oppose having +5% performance in the out-of-the-box installation :) But use your judgement on how much hassle would dealing with it be for Solus, given that some users would prefer anyway to build GROMACS anyway from source to get GPU acceleration.